pkg/apis/externaldns/types.go

```diff
@@ -194,6 +198,8 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("publish-internal-services", "Allow external-dns to publish DNS records for ClusterIP services (optional)").BoolVar(&cfg.PublishInternal)
 	app.Flag("publish-host-ip", "Allow external-dns to publish host-ip for headless services (optional)").BoolVar(&cfg.PublishHostIP)
 	app.Flag("connector-source-server", "The server to connect for connector source, valid only when using connector source").Default(defaultConfig.ConnectorSourceServer).StringVar(&cfg.ConnectorSourceServer)
+	app.Flag("generic-source-crd-apiversion", "Api version of the crd for generic source, format (group/version) valid only when using generic source").Default(defaultConfig.GenericSourceCRDApiVer).StringVar(&cfg.GenericSourceCRDApiVer)
```

Comment:
It might be helpful to have an example in the flag description like `test.k8s.io/v1alpha1` as you have used in the test.

Reply:
Suggest also capitalizing `API` and `CRD`.

Reply:
ok, would update.